blackbox_solver/Cargo.toml

```diff
@@ -32,6 +32,28 @@ p256 = { version = "0.11.0", features = [
     "arithmetic",
 ] }
 
+# Barretenberg WASM dependencies
+rust-embed = { version = "6.6.0", features = [
+    "debug-embed",
+    "interpolate-folder-path",
+    "include-exclude",
+] }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+wasmer = { version = "3.3", default-features = false, features = [ "js-default" ] }
+getrandom = { version = "0.2", features = [ "js" ]}
+wasm-bindgen-futures = "0.4.36"
+js-sys = "0.3.62"
+
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+getrandom = "0.2"
+wasmer = "3.3"
```

Comment:
While debugging this in acvm-backend-bb in the past, I found that it doesn't work quite right. All features for all dependencies are always activated for every target (it seems that it joins them all instead of filtering via the cfg). You probably need to toggle features with other feature flags 🙈